source/IdentityServer3.Localization/Properties/AssemblyInfo.cs

```diff
@@ -33,3 +34,5 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
+
+[assembly: InternalsVisibleTo("Unittests")]
```

Comment:
Sorry, I don't allow testing of internals. The unit tests are only allowed thru what's public. :/